c-api/iterator.po

```diff
@@ -26,39 +26,45 @@ msgid ""
 "sentinel value, calling the callable for each item in the sequence, and "
 "ending the iteration when the sentinel value is returned."
 msgstr ""
-"Python fournit deux itérateurs génériques. Le premier est un itérateur de "
-"séquence, il fonctionne avec n'importe quelle séquence gérant la méthode :"
-"meth:`__getitem__`. Le second fonctionne avec un objet appelable et une "
-"valeur sentinelle, il appelle l'appelable pour obtenir chaque élément de la "
-"séquence, et l'itération se termine lorsque la sentinelle est reçue."
+"Python fournit deux itérateurs d'usage générique. Le premier est un "
+"itérateur de séquence, il fonctionne avec n'importe quelle séquence gérant "
+"la méthode :meth:`__getitem__`. Le second fonctionne avec un objet appelable "
+"et une valeur sentinelle, l'appelable permet d'obtenir chaque élément de la "
+"séquence, et l'itération se termine lorsque la sentinelle est atteinte."
 
 #: ../Doc/c-api/iterator.rst:17
 msgid ""
 "Type object for iterator objects returned by :c:func:`PySeqIter_New` and the "
 "one-argument form of the :func:`iter` built-in function for built-in "
 "sequence types."
 msgstr ""
+"Type de l'itérateur renvoyé par les fonctions :c:func:`PySeqIter_New` et :"
+"func:`iter` à un seul argument."
 
 #: ../Doc/c-api/iterator.rst:24
 msgid "Return true if the type of *op* is :c:data:`PySeqIter_Type`."
-msgstr ""
+msgstr "Renvoie vrai si *op* est de type :c:data:`PySeqIter_Type`."
 
 #: ../Doc/c-api/iterator.rst:29
 msgid ""
 "Return an iterator that works with a general sequence object, *seq*.  The "
 "iteration ends when the sequence raises :exc:`IndexError` for the "
 "subscripting operation."
 msgstr ""
+"Renvoie un itérateur de séquence *seq*. L'itération prend fin lorsque la "
```

Comment:
> Renvoie un itérateur de séquence *seq*. 

Cette phrase ne me paraît pas donner toues les informations, je ne voit pas que l'itérateur renvoyé itère *seq*.